recipes/spirv-tools/all/conanfile.py

```diff
@@ -44,10 +44,15 @@ def configure(self):
             tools.check_min_cppstd(self, 11)
 
     def requirements(self):
-        self.requires("spirv-headers/{}".format(self._get_compatible_spirv_headers_version()))
+        self.requires("spirv-headers/{}".format(self._get_compatible_spirv_headers_version))
 
+    @property
     def _get_compatible_spirv_headers_version(self):
-        return "1.5.3" if tools.Version(self.version[1:]) >= "2020.2" else "1.5.1"
+        return {
+            "v2019.2": "1.5.1",
+            "v2020.3": "1.5.3",
+            "v2020.5": "1.5.4",
+        }.get(str(self.version))
```

Comment:
A consumer can always down- or upgrade the spirv-tools dependency in his profile or build recipe.
Should we also check whether the actual used spirv-headers is ok in `build`?

I had to do something similar in uvw but in package_info as it was header_only.

Reply:
Yes, https://github.com/conan-io/conan/pull/8053 should provide a better method for these checks.